gradle/libs.versions.toml

```diff
@@ -1,5 +1,5 @@
 [versions]
-kotlin = "1.9.24"
```

Comment:
The plan is to update it to "2.0.0" once it released, merge the branch and release kotlinx-io "0.4.0"